src/main/resources/petriNets/engine-processes/filter.xml

```diff
@@ -206,6 +210,7 @@
         <i18nString name="missing_allowed_nets">Chýbajúce procesy</i18nString>
         <i18nString name="missing_allowed_nets_desc">Zoznam chýbajúcich procesov pre aktuálny filter</i18nString>
         <i18nString name="parent_filter_id">ID rodičovského filtra</i18nString>
+        <i18nString name="allowed_nets">Povolené siete</i18nString>
     </i18n>
     <i18n locale="de">
```

Comment:
Missing translation of `allowed_nets` title

Reply:
Will correct it.